__mocks__/googleapis/index.ts

```diff
@@ -20,7 +20,7 @@ export namespace google {
       }
       getToken = () => ({tokens: []});
       // In dev-live, redirect straight to the reciever page.
-      generateAuthUrl = () => "http://localhost:3001/gauth?code=MockedCode"
```

Comment:
Either of these *should* work.  However, the fix is the technically correct version